test/dock_test.rb

```diff
@@ -0,0 +1,55 @@
+require './lib/boat'
+require './lib/renter'
+require './lib/dock'
+require 'minitest/autorun'
+require 'minitest/pride'
+
+class DockTest < Minitest::Test
+  def setup
+    @dock = Dock.new("The Rowing Dock", 3)
+    @kayak_1 = Boat.new(:kayak, 20)
+    @kayak_2 = Boat.new(:kayak, 20)
+    @canoe = Boat.new(:@canoe, 25)
+    @sup_1 = Boat.new(:standup_paddle_board, 15)
+    @sup_2 = Boat.new(:standup_paddle_board, 15)
+    @patrick = Renter.new("Patrick Star", "4242424242424242")
+    @eugene = Renter.new("Eugene Crabs", "1313131313131313")
+
+  end
+
+  def test_it_exists
```

Comment:
Since you are testing the `Dock` class, you don't need to retest the other classes.